press/api/client.py

```diff
@@ -115,6 +117,37 @@ def get_list(
 				.where(ParentDocType.team == frappe.local.team().name)
 			)
 
```

Comment:
also disallow get_list for Team doctype?

Reply:
https://github.com/frappe/press/commit/5b044baa3f5e04f34589f6006c0902434f81faca